client/db/types.go

```diff
@@ -662,12 +662,12 @@ func decodeAccountBackup(b []byte) (*AccountBackup, error) {
 // Save persists an account backup to file.
 func (ab *AccountBackup) Save(path string) error {
 	backup := ab.Serialize()
-	return ioutil.WriteFile(path, backup, 0o600)
+	return os.WriteFile(path, backup, 0o600)
```

Comment:
o for octal? I don't think I had ever seen this before.

Reply:
Yep. And the `o` is optional, so `0600` is also octal.